### PR TITLE
Feature/signup

### PR DESCRIPTION
**Pull request by guilhermemamprin**
Implements #9

@lucasBgabriel

### CODE REVIEW

File: src/app/about/about.controller.ts [hunk 1]
```diff
@@ -1,108 +1,108 @@
-module webShoes {
-  'use strict';
+// module webShoes {
+//   'use strict';
```
Comment (adrianovalente) [inline src/app/about/about.controller.ts]: Nice haha

Comment (adrianovalente) [inline src/app/about/about.controller.ts]: Apart from the commented code, 👍🏼

**Comment by adrianovalente** [comment]
Apart from the commented code, 👍🏼

File: src/app/about/about.controller.ts [hunk 2]
```diff
@@ -1,108 +1,108 @@
...
-    }
-  }
+//       awesomeThings.forEach(function(awesomeThing: Thing) {
+//         vm.awesomeThings.push(awesomeThing);
+//       });
+//     }
+//   }
 
-}
+// }
```
Comment (lucasBgabriel) [inline src/app/about/about.controller.ts]: Shouldn't be better if you delete this controller?

Comment (guilhermemamprin) [inline src/app/about/about.controller.ts]: Yes, it is quite useless. Will do hahah

**Pull request merged**